### ISSUE TITLE
autodoc choking on openff-units unit objects

### ISSUE THREAD

**Issue by richardjgowers** [OpenFreeEnergy/openfe#222]
Building the docs gets the following error (locally and on rtd):

```
WARNING: autodoc: failed to import class 'setup.LigandAtomMapper' from module 'openfe'; the following exception was raised:
Traceback (most recent call last):
  File "/home/richard/miniconda3/envs/openfe/lib/python3.9/site-packages/sphinx/ext/autodoc/importer.py", line 61, in import_module
    return importlib.import_module(modname)
  File "/home/richard/miniconda3/envs/openfe/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/richard/code/openfe/openfe/__init__.py", line 2, in <module>
    from . import setup
  File "/home/richard/code/openfe/openfe/setup/__init__.py", line 4, in <module>
    from gufe import (
  File "/home/richard/code/gufe/gufe/__init__.py", line 9, in <module>
    from .components import (
  File "/home/richard/code/gufe/gufe/components/__init__.py", line 5, in <module>
    from .solventcomponent import SolventComponent
  File "/home/richard/code/gufe/gufe/components/solventcomponent.py", line 15, in <module>
    class SolventComponent(Component):
  File "/home/richard/code/gufe/gufe/components/solventcomponent.py", line 34, in SolventComponent
    ion_concentration: unit.Quantity = 0.0 * unit.molar):
TypeError: unsupported operand type(s) for *: 'float' and 'molar'
```

**Comment by dwhswenson** [comment]
Transfer issue to gufe? Seems a gufe problem to me.

**Comment by richardjgowers** [comment]
it happens in the openfe repo, so it would need fixing here.  we masquerade the gufe objects as belonging to openfe (i.e. `openfe.setup.SmallMoleculeComponent` exists and is how we document it in notebooks) so they're also in the API docs too

**Comment by IAlibay** [comment]
@richardjgowers @dwhswenson this is the same issue we're now facing again (or maybe we alwasy faced it and it never got fixed?)

**Comment by IAlibay** [comment]
afaik we fixed this in #288
